Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxInputDateRangeDoc/HxInputDateRange_Documentation.razor

```diff
@@ -36,4 +36,9 @@
         (<code>Enabled="false"</code>) or set specific CSS class (<code>CssClass="...")</code>.
     </p>
     <Demo Type="typeof(HxInputDateRange_Demo_CalendarDateCustomization)" />
+
+    <DocHeading Title="Calendar Timezone" Level="3" />
```

Comment:
Unify "Timezone" vs. "TimeZone" vs. "..."
The correct term should be "Time zone" with a space, I think.